internal/resources/grafana/resource_alerting_rule_group.go

```diff
@@ -519,6 +542,15 @@ func packAlertRule(r *models.ProvisionedAlertRule) (interface{}, error) {
 		json["record"] = record
 	}
 
+	// FIXME: open api needs to be a reference to the duration
```

Comment:
@alexander-akhmetov I forgot to about this discovery. The `For` in the spec is a *Duration and the KeepFiringFor is a Duration. I think effectively they're the same thing but not 100% sure, can you have another look at this?

Reply:
I'm not sure why it's not a `*Duration` as well, they are [similarly defined](https://github.com/grafana/grafana/blob/5a10e3b43e1936fa20552bb2772cac4ceb83ac6f/pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go#L158-L163). I'll check.

Reply:
Probably because the For field is required, it's a pointer: https://github.com/go-swagger/go-swagger/issues/1386#issuecomment-360341120